data/rtaylor2D-q3.mesh

```diff
@@ -0,0 +1,1409 @@
+MFEM mesh v1.0
```

Comment:
That's one cool mesh :)

![GLVis_s01](https://user-images.githubusercontent.com/1278247/63983132-3d03f080-ca7a-11e9-8ff0-e06ed872a16b.png)